### DISCUSSION TITLE
How to review the wrong questions?

### DISCUSSION THREAD

**Discussion by aakashnand** [v-fidelusaleksander/ghcertified#188] [Study Together!]
On the last page when results are displayed. Question number is not shown. That makes it difficult to keep track of wrong questions.

When I click the question, I could see the question number then I check the corresponding question number markdown file but it was not matching. Are you shuffling the questions when test is started?

Is there option to store the history in local browser and clear the history?

For example this was my wrong question.

![image](https://github.com/FidelusAleksander/ghcertified/assets/14219201/de9fb39d-8cc6-45b5-9459-bd03b5c3b177)

But question number 37 from this file is different https://github.com/FidelusAleksander/ghcertified/blob/master/content/questions/foundations/question-037.md

**Comment by v-fidelusaleksander** [comment]
Hey @aakashnand 

Yes, when using the practice exams the questions are shuffled and the numbers do not correspond to the question numbers in the `Contributed questions` section

To review a question you have 2 options:

## After finished practice test (recommended)
At the end of the exam, you can click on the incorrect question and it will bring you back to that question
![image](https://github.com/FidelusAleksander/ghcertified/assets/63016446/fa720f9a-89cb-418c-a3e7-c2712e90c7f6)

I recommend reviewing all failed questions individually after the exam

## Using search
You can use the search tool to find the question you are looking for
![image](https://github.com/FidelusAleksander/ghcertified/assets/63016446/5d5d6051-737b-4407-9e7c-2f2268c8d66f)
